Common/TableProducer/trackPropagation.cxx

```diff
@@ -93,16 +97,34 @@ struct TrackPropagation {
     if (!o2::base::GeometryManager::isGeometryLoaded()) {
       ccdb->get<TGeoManager>(geoPath);
     }
+    if (checkGRPObject) {
+      ccdb->setFatalWhenNull(false);
+    }
   }
 
   void initCCDB(aod::BCsWithTimestamps::iterator const& bc)
   {
     if (runNumber == bc.runNumber()) {
       return;
     }
-    grpo = ccdb->getForTimeStamp<o2::parameters::GRPObject>(grpPath, bc.timestamp());
-    LOGF(info, "Setting magnetic field to %d kG for run %d from its GRP CCDB object", grpo->getNominalL3Field(), bc.runNumber());
-    o2::base::Propagator::initFieldFromGRP(grpo);
+    if (checkGRPObject) {
+      grpo = ccdb->getForTimeStamp<o2::parameters::GRPObject>(grpPath, bc.timestamp());
+      if (!grpo) {
+        grpmag = ccdb->getForTimeStamp<o2::parameters::GRPMagField>(grpmagPath, bc.timestamp());
+        if (!grpmag) {
+          LOG(fatal) << "Got nullptr from CCDB for path " << grpmagPath << " of object GRPMagField and " << grpPath << " of object GRPObject for timestamp " << bc.timestamp();
+        }
+        LOGF(info, "Setting magnetic field to current %d A for run %d from its GRPMagField CCDB object", grpmag->getL3Current(), bc.runNumber());
```

Comment:
I would use the "detail" level for log by default, unless you really care about having the message during real production.

Reply:
I agree, however this is the same level of info that was present before, this is also important information that should be printed once per run i.e. once per job ideally